### PR TITLE
Users

### PR DESCRIPTION
**Pull request by EricPudney**
[added encryption to users.api, renamed models consistently]

### CODE REVIEW

File: api/users.js
Comment (FelixGustafsson) [inline api/users.js:1]: We should send statuscodes along with the response for better errorhandling.

**Review by FelixGustafsson — CHANGES_REQUESTED** [review]
We should send statuscodes along with the response for better errorhandling. (api/users.js)

File: api/users.js
Comment (EricPudney) [reply api/users.js:1]: absolut, jag fixar imorgon

Comment (EricPudney) [reply api/users.js:1]: jag tror det blir bra nu.

**Review by FelixGustafsson — APPROVED** [review]

**Review by VeroniccaStrand — APPROVED** [review]

**Pull request merged**